thoth/package_extract/core.py

```diff
@@ -69,7 +69,7 @@ def extract_image(image_name: str, timeout: int = None, *,
         result = run_analyzers(rootfs_path)
         result['layers'] = layers
 
-    push_gateway = os.getenv('PROMETHEUS_PUSH_GATEWAY', 'pushgateway:9091')
+    push_gateway = os.getenv('PROMETHEUS_PUSHGATEWAY_HOST:PROMETHEUS_PUSHGATEWAY_PORT')
```

Comment:
This will not work. You are requesting an environment variable that is called "``PROMETHEUS_PUSHGATEWAY_HOST:PROMETHEUS_PUSHGATEWAY_PORT``"